run-tests.js

```diff
@@ -8,7 +8,13 @@ var mocha = new Mocha()
 
 var testDir = 'test'
 require('babel/register')({
-  ignore: false
+  ignore: function (filename) {
+    if (filename.indexOf('weakee') > -1) {
+      return false
+    }
+
+    return filename.indexOf('node_modules') > -1
```

Comment:
this is kinda weird - we need Babel to transpile the weakee module (which in itself is weird - I'd expect that module to publish its transpiled version) - but if we make Babel transpile everything I get a weird error from Chai when trying to use the `to.throw` assertion (see https://github.com/chaijs/chai/issues/384 for more). This updates Babel to not transpile any node_modules except weakee.